src/instructlab/model/convert.py

```diff
@@ -45,7 +45,6 @@
     show_default=True,
 )
 @utils.macos_requirement(echo_func=click.secho, exit_exception=click.exceptions.Exit)
-@click.pass_context
```

Comment:
Why?

Reply:
The reason I removed it because `ctx` is not used and pylint reports warning/error. But another PR was merged before mine which suppress pylint for `ctx`, which is a quick but not good fix. If the argument is not used, we should not leave it in codes. 

`ctx` should be removed. I'll submit a change. @leseb